### DISCUSSION TITLE
about/

### DISCUSSION THREAD

**Discussion by giscus** [hellowac/hellowac.github.io#4] [Announcements]
# about/

hellowac的工作学习生活笔记

https://hellowac.github.io/about/

**Comment by shenweiyan** [comment]
又看到一个基于 mkdocs-material 的博客，真不错 👍

**Comment by KingPuiWong** [comment]
你好 大佬，有考虑开源 pptx 转 json 和 json 转回 pttx 的项目吗？或者是否可以写些文章分享一下~

**Reply by hellowac** [reply]
现在业务不在这一块喔，没有时间和精力搞。你可以参考我之前花大时间和精力搞的 [ecma-376中文版](https://github.com/hellowac/ecma-376-zh-cn)的文档，你把这个文档吃透了，就可以自己写轮子解析pptx/docx文件了。
